jiant/models.py

```diff
@@ -946,8 +968,17 @@ def _seq_gen_forward(self, batch, task, predict):
         sent, sent_mask = self.sent_encoder(batch["inputs"], task)
         out["n_exs"] = get_batch_size(batch)
 
+        if isinstance(task, (CharSeq2SeqTask)):
+            decoder = getattr(self, "%s_decoder" % task.name)
```

Comment:
something seems weird... should all sequence generation tasks have a decoder ?

Could be legacy code that should have been deleted earlier. In any case, it should be okay if we delete the `if` statement

Reply:
Yeah, second this